core/meta/src/TClass.cxx

```diff
@@ -1411,6 +1411,7 @@ void TClass::Init(const char *name, Version_t cversion,
          }
       }
       fClassInfo = gInterpreter->ClassInfo_Factory(givenInfo);
```

Comment:
These two are the only lines about which we did not reach consensus at the PPP today. The open question is whether we need to put them in a critical section or if it is enough to take the lock in LoadClassInfo.